### DISCUSSION TITLE
Integrating Native-Base

### DISCUSSION THREAD

**Discussion by hazhayder** [mmazzarolo/react-native-universal-monorepo#31] [Q&A]
Hi, 
First of all kudos to the maintainer you've done an awesome job. We're  trying to integrate Native-Base in this project but we want this to reside under "my-app/app" space so we could share components based on native-base between web and mobile.  After installing native-base under mentioned space "my-app/app" it says that cannot find module react-native 
![image](https://user-images.githubusercontent.com/12206002/141693127-6b667fed-92ab-4f1f-82f6-44ee57beea46.png)
**/packages/app/src/components/Button.js**
```
// import React from "react"
import { Button, Center, NativeBaseProvider } from "native-base"
function Button(){
  return (
    <>
      <Button onPress={() => console.log("hello world")}>Primary</Button>
    </>
  )
}
export default Button;
```

**Comment by mmazzarolo** [comment]
Converted to discussion since it's not an issue 👍

**Reply by hazhayder** [reply]
Hi @mmazzarolo Thanks for replying do you've something to share?
